app/grandchallenge/algorithms/templates/algorithms/algorithm_detail.html

```diff
@@ -60,10 +60,21 @@
             {% endif %}
 
             {% if "execute_algorithm" in algorithm_perms and object.latest_ready_image %}
+                {% if object.use_flexible_inputs %}
+                    <a class="nav-link"
+                       href="{% url 'algorithms:execution-session-create-new' slug=object.slug %}">
+                        <i class="fas fa-file-import fa-fw"></i>&nbsp;Try-out Algorithm with flexible inputs (experimental!)
+                    </a>
+                {% else %}
                 <a class="nav-link"
-                   href="{% url 'algorithms:execution-session-create' slug=object.slug %}">
-                    <i class="fas fa-file-import fa-fw"></i>&nbsp;Try-out Algorithm
-                </a>
+                       href="{% url 'algorithms:execution-session-create' slug=object.slug %}">
+                        <i class="fas fa-file-import fa-fw"></i>&nbsp;Try-out Algorithm
+                    </a>
+                {% endif %}
+            {% endif %}
+
+            {% if request.user.is_staff and "execute_algorithm" in algorithm_perms and object.latest_ready_image %}
+
             {% endif %}
```

Comment:
```suggestion
```

Reply:
Nice catch, I think something went wrong when automerging Miriam's changes that I didn't have locally yet.

Reply:
weird, the is_staff has been removed already?